frontend/device/kindle/device.lua

```diff
@@ -354,6 +353,17 @@ function KindleOasis:init()
 
     self.input.open(self.touch_dev)
     self.input.open("/dev/input/by-path/platform-gpiokey.0-event")
+
+    -- get rotate dev by EV=d
+    local std_out = io.popen("cat /proc/bus/input/devices | grep -e 'Handlers\\|EV=' | grep -B1 'EV=d'| grep -o 'event[0-9]'", "r")
```

Comment:
Doesn't the bma acceleremoter get a recognizable path somewhere (input/by-path?)?